js/interface.js

```diff
@@ -2383,12 +2383,14 @@ function getSecurityRule() {
 
   if (currentFinalRules.length > 0) {
     currentFinalRules.forEach(function(rule) {
-      if (widgetData.tokenId) {
-        if (rule.allow.hasOwnProperty('tokens') && Number(rule.allow.tokens[0]) === widgetData.tokenId) {
+      if (rule.allow.hasOwnProperty('tokens')) {
```

Comment:
Use early exit clauses

Reply:
I removed this code because of efficient code